Test/Unit/Scope/VariableTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Marsskom\Generator\Test\Unit\Scope;
+
+use Marsskom\Generator\Api\Data\Scope\ScopeInterface;
+use Marsskom\Generator\Exception\Scope\VariableAlreadySetException;
+use Marsskom\Generator\Exception\Scope\VariableIsNotMultipleException;
+use Marsskom\Generator\Exception\Scope\VariableNotExistsException;
+use Marsskom\Generator\Exception\Scope\VariableTypeException;
+use Marsskom\Generator\Model\Enum\InputParameter;
+use Marsskom\Generator\Test\Unit\Mock\Sequence\FakeContextRegister;
+use Marsskom\Generator\Test\Unit\MockHelper\ScopeFactory;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
+
+class VariableTest extends MockeryTestCase
+{
+    private ScopeInterface $scope;
+
+    /**
+     * @inheritdoc
+     */
+    protected function mockeryTestSetUp()
+    {
+        $this->scope = (new FakeContextRegister())->execute((new ScopeFactory())->create([
```

Comment:
Fix the code style, because it is not clearly to understand that `->create` belongs to `ScopeFactory`.